django/db/utils.py

```diff
@@ -118,11 +118,9 @@ def load_backend(backend_name):
             if ispkg and name not in {'base', 'dummy', 'postgresql_psycopg2'}
         ]
         if backend_name not in ['django.db.backends.%s' % b for b in builtin_backends]:
-            backend_reprs = map(repr, sorted(builtin_backends))
             raise ImproperlyConfigured(
-                "%r isn't an available database backend.\n"
-                "Try using 'django.db.backends.XXX', where XXX is one of:\n"
```

Comment:
Maybe we should keep this and rephrase to "To use one of the built-in backends, use 'django.db.backends.XXX', where...